apps/o2-blocks/package.json

```diff
@@ -19,9 +19,10 @@
 	"scripts": {
 		"clean": "npx rimraf dist",
 		"prebuild": "yarn run clean",
-		"build": "calypso-build editor='./src/editor.js' view='./src/view.js' --env.WP"
+		"build": "calypso-build"
```

Comment:
Providing two different entrypoints in the CLI is not longer supported (https://github.com/webpack/webpack-cli/issues/2258). Moved to a custom `webpack.config.js`.

Reply:
The `editing-toolkit` build has the following syntax for adding two entrypoints to the build:
```js
calypso-build --env source='src/editor','src/view'
```
The outcome is exactly the same.

It would be nice to add this functionality to the base `calypso-build` script. But that's out of scope for this PR. @noahtallen's #48896 and related work is a better fit.

Reply:
We want to fix it, in the next patch release we will return supporting of `webpack ./entry.js` and ``webpack entry=./entry.js`